channeldb/invoice_test.go

```diff
@@ -1013,6 +1013,53 @@ func TestSettleIndexAmpPayments(t *testing.T) {
 	require.Nil(t, err)
 }
 
+// TestFetchPendingInvoices tests that we can fetch all pending invoices from
+// the database using the FetchPendingInvoices method.
+func TestFetchPendingInvoices(t *testing.T) {
+	t.Parallel()
+
+	db, err := MakeTestInvoiceDB(t, OptionClock(testClock))
+	require.NoError(t, err, "unable to make test db")
+
```

Comment:
can we also add a case here to test the empty slice returned from `FetchPendingInvoices`?

Reply:
Yeah good idea! Done.